keps/sig-network/2593-multiple-cluster-cidrs/README.md

```diff
@@ -791,6 +791,12 @@ This section must be completed when targeting beta to a release.
 Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
+kube-controller-manager needs to be restarted, the CIDR allocator will be switched 
```

Comment:
Commenting here, as I comment on unchanged lines

L596/L605 - please add links to the tests that already exist (and their k8s-triage) as requested in the template:
https://github.com/kubernetes/enhancements/blame/master/keps/NNNN-kep-template/README.md#L318-L325


L777: Have those tests been added? Can you please link?

Reply:
> L596/L605 - please add links to the tests that already exist (and their k8s-triage) as requested in the template:
https://github.com/kubernetes/enhancements/blame/master/keps/NNNN-kep-template/README.md#L318-L325

Added links to the integration tests, however not sure which link I should include for the k8s-triage
Tried the following: https://storage.googleapis.com/k8s-triage/index.html?date=2023-01-01&pr=1&test=TestIPAMMultiCIDRRangeAllocatorCIDRAllocate which is for one of the integration tests, however it returns 

```
0 clusters of 0 failures out of 116076 builds from 12/17/2022, 4:00:06 PM to 1/1/2023, 2:53:24 PM.
```

Probably my query is wrong

> L777: Have those tests been added? Can you please link?

Planning to add them before beta

Reply:
> Planning to add them before beta

OK - please ensure that it really happens before we switch the feature gate to Beta in k/k.
@aojea ^^ - if you could take care of ensuring it (you will be probably reviewing the code...)

Reply:
This feature is covered with integration https://github.com/kubernetes/kubernetes/tree/master/test/integration/clustercidr because it is an API change that may or not be implemented by external components.
Since GKE is going to implement it, we plan to add additional coverage with e2e test that exercise the feature too